new-lamassu-admin/src/pages/Maintenance/MachineStatus.js

```diff
@@ -51,7 +53,20 @@ const MachineStatus = () => {
       width: 250,
       size: 'sm',
       textAlign: 'left',
-      view: m => m.name
+      view: m => (
+        <div className={classes.flexRow}>
+          {m.name}
+          <div
+            className={classes.machineRedirectContainer}
+            onClick={() => {
+              history.push(`/machines/${m.deviceId}`, {
+                selectedMachine: m.name
+              })
```

Comment:
Do we still need this selectedMachine thing? Isn't the ID enough?

Reply:
Hmm, I tried removing it and nothing got broken, so I guess it can go